services/grip/frontend/public/test_schemas.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
Do we want to commit this under public? 
If it's not needed for the app might not want it. Could be useful test data under some test/fixtures directory maybe?

Reply:
This file is just temporary. I need it for the demo of the sprint review then I'll blow it away.
Do you want me to add it to the .gitignore?

Reply:
Yeah I'd leave it locally if it's just for a specific demo session on one local machine. But it's useful to have test data for unit tests and other things so could consider moving it to a test folder location too.

Reply:
Do you want test/fixtures to be at the top level @loneil ? Or within Grip?